satnogsclient/__init__.py

```diff
@@ -21,6 +21,3 @@
 
     if not GROUND_STATION_LON:
         raise Exception('GROUND_STATION_LON not configured')
-
-    if not GROUND_STATION_ELEV:
```

Comment:
"if not" also fails if the variable is 0, where elevation 0m is still a feasible value.